sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -2065,29 +2065,238 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
     }
   }
 
-  def testLineSeparator(lineSep: String): Unit = {
```

Comment:
You don't have to completely remove this out .. just take out the ones not working.